### PR TITLE
Add option to mark labs as inactive 

### PR DESCRIPTION
**Pull request by k-nut**
Fixes [#401](https://github.com/okfde/codefor.de/issues/401)

### CODE REVIEW

**Review by ubergesundheit — APPROVED** [review]

File: themes/codefor-theme/layouts/partials/map-overview.html [hunk 1]
```diff
@@ -47,39 +47,49 @@
...
-  {{- dict "coords" $coords "label" $label "link" .RelPermalink "city" .Params.city | jsonify | safeJS -}},
+  {{- dict "coords" $coords "label" $label "link" .RelPermalink "city" .Params.city "inactive" .Params.inactive | jsonify | safeJS -}},
 {{- end -}}
 ]
 
   var labsLayer = L.featureGroup().addTo(map);
 
   labs.forEach(function (lab) {
+    const a = document.createElement('a');
+    const suffix = lab.inactive ? " (Inaktiv)" : "";
```
Comment (sonjafischbauer) [inline themes/codefor-theme/layouts/partials/map-overview.html:86]: Please remove (or do not add, in the first place) the word "Inaktiv" to the grey marker on the map. 

The indication of whether a lab is currently active or not should be only the color of the marker. 

Reason: We have quite a few labs which are inactive at any given moment. While this is fine and normal, I think that displaying it bluntly on the map, which is often the first thing that people see, would look discouraging to people who want to join the community.

File: themes/codefor-theme/layouts/labs/single.html [hunk 2]
```diff
@@ -13,6 +13,13 @@ <h1 class="headline-brackets blue flex-grow-1 m-0">{{.Title}} </h1>
     {{ partial "hero-block-center.html" (dict "context" . "txt" .Params.Hero ) }}
     {{ end }}
 
+    {{ if .Params.inactive }}
+        <div class="alert alert-warning mt-4" role="alert">
+            Dieses Lab ist nicht mehr aktiv. <br />  Wenn du Fragen zum Netzwerk hast, dann schreib
+            uns gerne direkt an info@codefor.de.
+        </div>
+    {{ end }}
```
Comment (sonjafischbauer) [inline themes/codefor-theme/layouts/labs/single.html:21]: Please change text to: 

Dieses Lab ist gerade nicht aktiv. Das bedeutet, dass aktuell keine Treffen stattfinden und die Kontaktpersonen ihr Ehrenamt derzeit ruhen lassen. Wenn du Fragen zum Netzwerk hast, dann schreib uns gern an info@codefor.de.

**Review by sonjafischbauer — DISMISSED** [review]
thank you so much! 
we will mark labs inactive with separate pull requests.

**Comment by sonjafischbauer** [comment]
lab_marker_inactive_2

 After my review was already done, I thought more about accessibility, and that the markers in grey and red might be hard to tell apart for someone who can not see color (red and grey = similar contrast). 
 
 So I created an alternative for the inactive lab svg

File: themes/codefor-theme/layouts/labs/single.html [hunk 3]
```diff
@@ -13,6 +13,13 @@ <h1 class="headline-brackets blue flex-grow-1 m-0">{{.Title}} </h1>
     {{ partial "hero-block-center.html" (dict "context" . "txt" .Params.Hero ) }}
     {{ end }}
 
+    {{ if .Params.inactive }}
+        <div class="alert alert-warning mt-4" role="alert">
```
[the remark this answers is not in the document]
Comment (k-nut) [reply themes/codefor-theme/layouts/labs/single.html]: Changed ✅

File: themes/codefor-theme/assets/scss/main.scss [hunk 4]
```diff
@@ -51,7 +51,7 @@ $codefor-light-grey: #F2F2F2;
 @import "bootstrap/pagination";
 //@import "bootstrap/badge";
 //@import "bootstrap/jumbotron";
-//@import "bootstrap/alert";
+// @import "bootstrap/alert";
```
Comment (k-nut) [inline themes/codefor-theme/assets/scss/main.scss:54]:
```suggestion
//@import "bootstrap/alert";
```

**Pull request merged**